samples/scala-protobuf-tracing/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```

Comment:
I didn't even know that 1.9.9 existed. 

Maybe in follow-up PR we can upgrade the other samples.

Reply:
https://github.com/lightbend/kalix-jvm-sdk/issues/2075